packages/_private/e2e-demo-test/src/test.page.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @copyright
+ * Copyright(c) 2020 SICK AG
+ */
+
+import { Locator, Page, Response } from '@playwright/test';
+import locators from './test.selector';
```

Comment:
I think the naming here is kind of off. The `test.selector` file includes the selectors. This is also seen in the types of `locator`, which uses a `selector` as its first argument. Suggestion: Rename potentially misleading import declaration `locators` and rename it `selectors`.

```suggestion
import selectors from './test.selector';
```